vision_datasets/common/data_manifest/data_manifest.py

```diff
@@ -199,3 +199,82 @@ def __eq__(self, other) -> bool:
 
     def __len__(self):
         return len(self.images)
+
+
+class AnnotationDataManifest(ManifestBase):
+    """
+    Encapsulates information of an annotation. The annotation can be associated with a single image, or multi-image jointly. 
+    """
+
+    def __init__(self,
+                 id: int,
+                 img_ids: List[int],
+                 label: ImageLabelManifest,
```

Comment:
Initially, I just used ImageLabelManifest since it serves my goal except that the name is related to `image`. However, combined with Julia's suggestion, I created a new class `MultiImageLabelManifest` which inherits `ImageLabelManifest` and add `img_ids` and `id` as additional information. I also renamed `AnnotationWiseDatasetManifest` as `MultiImageDatasetManifest`, which contains information of `list[ImageDataManifest` `list[MultiImageLabelManifest`